central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -372,37 +391,39 @@ func (s *flowStoreImpl) retryableRemoveFlowsForDeployment(ctx context.Context, i
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
 
+	// To avoid a full scan with an OR delete source and destination flows separately
```

Comment:
Should `RemoveFlowsForDeployment` be retryable?  I think if we do not make it retryable then a subsequent prune will catch back up.  Thoughts?

Reply:
What is the concern if we make it retryable? If we make it retriable, the Dst maybe removed before Src. If the failure is caused by timeout, we may have a little better chance to get all pruning done.

Reply:
The order of removal does not matter. We are removing the complete flow where the orphaned deployment is either the source or the destination.